Frontend/ui-driver/src/Flow.purs

```diff
@@ -2144,6 +2144,7 @@ clearPendingDuesFlow :: Boolean -> FlowBT String Unit
 clearPendingDuesFlow showLoader = do
   void $ lift $ lift $ toggleLoader showLoader
   liftFlowBT $ runEffectFn1 initiatePP unit
+  liftFlowBT $ runEffectFn1 initiatePPSingleInstance unit
```

Comment:
both initiatePP and initiatePPSingleInstance will be required to called everytime? If yes, then good to have a wrapper which call both together